.travis.yml

```diff
@@ -36,8 +36,6 @@ install:
 before_script:
   - echo "zend_extension=xdebug.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
 
-script:
-  - vendor/bin/phpunit -c phpunit.xml.dist --coverage-clover build/logs/clover.xml
+script: ./vendor/bin/phpunit tests/YahooWeatherClientTests.php --coverage-html --coverage-clover=coverage.xml
```

Comment:
If U mainly PHPUnit i recommand an composer global install of the library. So you will only type `phpunit` 😉